app/src/main/res/layout-land/onboarding_slide.xml

```diff
@@ -46,24 +47,22 @@
         android:gravity="center_horizontal"
         android:text="@{viewModel.title}"
         android:textColor="@color/oppiaPrimaryText"
```

Comment:
I think this color is part of `Heading1`, if so can you remove this attribute here?

Reply:
Yes, I have now removed the ` android:textColor="@color/oppiaPrimaryText"` in the latest commit